Actions/DetermineArtifactUrl/README.md

```diff
@@ -7,19 +7,19 @@ Determines the artifactUrl to use for a given project
 | Name | Description |
 | :-- | :-- |
 | Settings | env.Settings must be set by a prior call to the ReadSettings Action |
+| Secrets | env.Secrets with insiderSasToken must be read by a prior call to the ReadSecrets Action |
```

Comment:
insiderSasToken can be missing, in which case env:Secrets.insiderSasToken is empty - and it will work with bcartifacts.
But the action requires that you have called ReadSecrets with insiderSasToken for Secrets.insiderSasToken to exist.